app/src/main/java/org/bspb/smartbirds/pro/db/NomenclatureColumns.java

```diff
@@ -29,4 +30,7 @@
     @DataType(TEXT)
```

Comment:
Can't comment it - there is a `label_bg` column above. Shouldn't that be renamed to `label_local`?